tests/wdio/application-status-overlay/application-status-overlay-spec.js

```diff
@@ -60,23 +62,23 @@ Terra.describeViewports('ApplicationStatusOverlay', ['large'], () => {
       browser.url('/raw/tests/terra-application/application-status-overlay/status-overlay-priority');
     });
 
-    it('renders with last registered status view', () => {
+    it('should render with last registered status view', () => {
       browser.click('#show-status');
-    });
 
-    Terra.it.validatesElement('initial', { selector: '#root' });
+      Terra.validates.element('initial', { selector: '#root' });
+    });
 
-    it('renders with same last registered status view even after updates to other status views', () => {
+    it('should render with same last registered status view even after updates to other status views', () => {
       browser.click('#button1');
       browser.click('#button2');
-    });
 
-    Terra.it.validatesElement('other status view updates', { selector: '#root' });
+      Terra.validates.element('other status view updates', { selector: '#root' });
+    });
 
-    it('renders with updated last registered status view', () => {
+    it('render with updated last registered status view', () => {
```

Comment:
nit: To be consistent with above.
```suggestion
    it('should render with updated last registered status view', () => {
```